### PR TITLE
[Fixes #310] Recommendation system for model layers

### LINKED ISSUE #310: Recommendation system for model layers
## Feature Request

Is your feature request related to an existing [Issue](https://www.github.com/Auto-DL/Generator/issues) or a new implementation?

New implementation

### Feature Description

Give user recommendation for next layer depending on the current model.

### PR DESCRIPTION
**Pull request by VedantMahadik**
Signed-off-by: VedantMahadik <therookie636@gmail.com>

# Pull Request

## What does this PR do?
Fixes #310
Shows top two recommendations for model layers, which gives users suggestions depending on the latest three layers added.

## What part does this affect?
- [x] FrontEnd.
- [ ] BackEnd.
- [ ] Documentation.
- [ ] Other. (Please specify below)

## Before submitting
- [x] Was this discussed/approved via a GitHub issue or slack?
- [x] Did you read the [contributor guideline](https://github.com/Auto-DL/Auto-DL/blob/v1-beta/CONTRIBUTING.md)?
- [x] Did you ensure that there aren't any other open [Pull Requests](https://www.github.com/Auto-DL/Generator/pulls) for the same update/change?
- [x] Did you make sure the title is self-explanatory and the description concisely explains the PR?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure the code is clean and docstrings have been added or updated as required?
- [ ] Did you make sure the code is linted/formatted locally prior to submission? (using `black` and/or `prettier`)
- [x] Did you make sure to update the documentation with your changes? (if necessary)

## PR review
Anyone in the community is free to review the PR once the tests have passed.

Thank you for contributing to [AutoDL](https://github.com/Auto-DL/Auto-DL). We look forward to your continued support.

### CODE REVIEW

**Review by bamblebam — APPROVED** [review]

File: FrontEndApp/v1-react/src/Components/MainApp/Step2.js [hunk 1]
```diff
@@ -2318,15 +2319,24 @@ function Step2() {
...
-  };
+    const recommendations = await getRecommendations(tempArr)
+		console.log(recommendations)
+	}
+
+	const getRecommendations = layersArr => {
+		let recommendationArr = layersArr.map(x => x.name)
+
+		return axios
+			.post("http://localhost:8080/predict", {
```
Comment (ADI10HERO) [inline FrontEndApp/v1-react/src/Components/MainApp/Step2.js:2330]:
```suggestion
			.post("http://localhost:8080/predict", {
```

Please use env variables.

**Review by ADI10HERO — CHANGES_REQUESTED** [review]

**Comment by ADI10HERO** [comment]
Add the following to frontend `.env` file: `REACT_APP_FASTAPI_URL`

**Comment by ADI10HERO** [comment]
@rajraj889 
The code looks okay but is it placed correctly? Or should the API call be in a service file and just a call to that function from here?

**Comment by rajraj889** [comment]
@ADI10HERO Yes it's a practice we are following, so it would be great if it's in service file.

**Comment by ADI10HERO** [comment]
@VedantMahadik  I tried resolving conflicts using the Github UI, but the 3rd Step2 conflict confused me, leaving this to you :P

**Comment by ADI10HERO** [comment]
@bamblebam  Reminder :)

**Comment by bamblebam** [comment]
Yeah I will solve them this week

**Comment by VedantMahadik** [comment]
Sorry for the delay, I have merged the branches please review it.

**Review by ADI10HERO — APPROVED** [review]

**Pull request merged**